quay/src/v1/mod.rs

```diff
@@ -1,5 +1,5 @@
 use failure::Fallible;
-use reqwest::{self, async};
+use reqwest::{self, r#async};
```

Comment:
You can turn this into
```suggestion
use reqwest::{self, r#async as asynchro};
```
and then use `asynchro::` everywhere else in the rest of this file.

Reply:
Maybe better to just remove this `use` here and change the code to `reqwest::async` where needed

Reply:
```suggestion
use reqwest;
```

(note that this will break the other code)

Reply:
`async` is a reserved keyword in edition-2018, which means that will end up with ` r#async` everywhere, thus why I suggested to rename-use as `asynchro` instead.

Reply:
Are keywords also reserved even if they're not used as a whole word? I thought that `reqwest::async` should be an issue.

Reply:
it is as Luca said so I will change to `use reqwest::{self, r#async as asynchro};`

Reply:
I can't confirm that. I tried my suggestion locally and it works.

Please see this diff: https://github.com/shiywang/cincinnati/compare/2018edition...steveeJ-forks:pr/upstream-104?expand=1

Reply:
okay, I think `cargo fix --edition` just replace `async` to `r#async` unconditionally, so rustc should be fine with this, I merged your pr, thanks!!